docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -77,9 +84,13 @@ the phase transitions look like this:
   and gotchas before going with semi-automatic version.
 </Admonition>
 
-It is possible to rotate a specific set of certificate authorities using the 
-`--type` flag. If this flag is not provided, then all certificate authorities
-will be rotated. 
+It is recommended to rotate a specific certificate authority using the 
```

Comment:
I think this paragraph would be clearer if we were to replace the passive voice ("It is recommended", "was not provided", etc.) with the active voice.

Reply:
I pushed up some changes. Let me know if that reads better.